test/e2e/load.go

```diff
@@ -70,7 +70,17 @@ var _ = Describe("Load capacity", func() {
 	framework.NamespaceDeletionTimeout = time.Hour
 
 	BeforeEach(func() {
-		c = framework.Client
+		// Explicitly create a client with higher QPS limits.
+		config, err := loadConfig()
+		Expect(err).NotTo(HaveOccurred())
```

Comment:
Could we take this out to a generic f(n) - I'm guessing it would be useful in other testing contexts.